doc/source/user_guide/timeseries.rst

```diff
@@ -1271,6 +1271,21 @@ frequencies. We will refer to these aliases as *offset aliases*.
     "U, us", "microseconds"
     "N", "nanoseconds"
 
+.. note::
+
+   When using the offset aliases above, it should be noted that functions
+   such as ``date_range``, ``bdate_range``, will only return timestamps that
```

Comment:
also if you are adding here it is ok, but would be great to add to the actual functions as well

Reply:
use something like ``` :func:`pd.date_range` ``` to reference this

Reply:
Correct sorry I missed this :func:, this is updated